lib/openbook.ts

```diff
@@ -249,21 +294,24 @@ export const isClosableOrder = (order: OpenOrdersAccountWithKey): boolean =>
   order.account.position.baseFreeNative.eq(BN_0) &&
   order.account.position.quoteFreeNative.eq(BN_0);
 
-export const isOpenOrder = (order: OpenOrdersAccountWithKey, markets: Markets): boolean => {
+export const isOpenOrder = (
```

Comment:
These should just be market utility functions.

Reply:
In that they don't need to exist here.

Reply:
Per discussion, if new SDK from openbook doesn't have it, move it into openbookMarkets.ts